src/lxc/tools/lxc_top.c

```diff
@@ -316,22 +378,29 @@ static void stats_print_header(struct stats *stats)
 static void stats_print(const char *name, const struct stats *stats,
 			const struct stats *total)
 {
-	char blkio_str[20];
+	char iosb_str[63];
+	char iosb_total_str[20];
+	char iosb_read_str[20];
+	char iosb_write_str[20];
 	char mem_used_str[20];
 	char kmem_used_str[20];
 	struct timeval time_val;
 	unsigned long long time_ms;
 
 	if (!batch) {
-		size_humanize(stats->blkio, blkio_str, sizeof(blkio_str));
+		size_humanize(stats->io_service_bytes.total, iosb_total_str, sizeof(iosb_total_str));
+		size_humanize(stats->io_service_bytes.read, iosb_read_str, sizeof(iosb_read_str));
+		size_humanize(stats->io_service_bytes.write, iosb_write_str, sizeof(iosb_write_str));
 		size_humanize(stats->mem_used, mem_used_str, sizeof(mem_used_str));
 
-		printf("%-18.18s %12.2f %12.2f %12.2f %14s %10s",
+		snprintf(iosb_str, sizeof(iosb_str), "%s(%s/%s)", iosb_total_str, iosb_read_str, iosb_write_str);
```

Comment:
Because the this function returns void, like function `size_humanize` in the same file. 

But it is better check the return value, so add the check  and just give a warning.